cmd/gazelle/fix-update.go

```diff
@@ -348,6 +350,7 @@ func runFixUpdate(wd string, cmd command, args []string) (err error) {
 		// Generate rules.
 		var empty, gen []*rule.Rule
 		var imports []interface{}
+		var relsToVisit []string
```

Comment:
Shouldn't this be a set to avoid duplicates?

Reply:
Or maybe it doesn't matter because this just gets passed along and `relsToVisitSeen` handles it later?

Reply:
Yeah, `relsToVisitSeen` will handle it later.

I thought about this, but I expect duplicates to be uncommon here (multiple extensions want to index the same directory), so allocating a map to deduplicate might cost more memory than it saves.